src/windows/Microsoft.Authentication.Helper.Windows/Application.cs

```diff
@@ -72,18 +61,45 @@ private static string GetArgument(IDictionary<string, string> inputDict, string
 
         protected virtual async Task<string> GetAccessTokenAsync(string authority, string clientId, Uri redirectUri, string resource)
         {
-            var cache = new VisualStudioTokenCache(Context);
-            var authContext = new AuthenticationContext(authority, cache);
+            string[] scopes = { $"{resource}/.default" };
```

Comment:
> what is the resource you are using

In practice we're using Azure DevOps which is only identified as a GUID.. so the scope here will be `00000-...-0000/.default`

Since there are no defined scopes on the Azure DevOps application (it's still a v1 app) appending the `/.default` I am told will request the default 'v1' scopes which is `user_impersonation`.

Reply:
I remember that when VS tested, they saw that AcquireTokenSilent never fetched the token from the cache, it always refreshed the RT. So it's a good test case to have.